wynnset/tapestry-wp
# Default Node Permissions and Access Tab Control

Closes #473 
#### Adds:
- Access tab to settings modal
- `defaultPermissions` settings field
- `viewAccess` settings field
- Node initialization with default permissions in both front and backend
- Access tab viewing control based on edit permissions and settings 
____
#### Steps to test:
Default Permissions:
 - Create a new node and verify the permissions, without modification, are set to `public: read` and `authenticated: read`.
 - Go to the settings modal and modify the default permissions in the Access tab
 - Create another node and verify the default permissions have changed

Show/Hide Access Tab:
 - Give public edit/add permission to a node
 - Open an incognito window and click to edit the node
 - Verify you can see the Access tab as an un-authenticated user 
 - Switch back to the logged in window and flip the switch in settings modal to disable Access tab viewing
 - Verify you (the author) can still see and modify the Access tab for any node while logged in
 - Log out/switch to incognito and click to edit that same node
 - Verify the Access tab is hidden
